### PR TITLE
Form re work

### PR DESCRIPTION
**Pull request by tomriiich**

### CODE REVIEW

File: src/components/Home.jsx [hunk 1]
```diff
@@ -8,11 +8,11 @@ function Home() {
...
         <img className="logoPer" src={Logo} alt="logo" />
-        <NavLink className="btnLog navLog" exact to="/login">
-          Login
-        </NavLink>
         <NavLink className="btnSign navSign" exact to="/register">
-          SignUp
+          S&apos;inscrire
+        </NavLink>
+        <NavLink className="btnLog navLog" exact to="/login">
+          Connexion
```
Comment (rocambille) [inline src/components/Home.jsx]: "S'inscrire" and "Se connecter", or "Inscription" and "Connexion": pick one

File: src/components/Register.jsx [hunk 2]
```diff
@@ -43,77 +44,80 @@ function Register() {
...
+              <label htmlFor="password">Mot de passe</label>
+              <input
+                type="password"
+                id="password"
+                name="password"
+                onChange={onChange}
+                value={user.password}
+                placeholder="********"
+              />
+            </div>
```
Comment (rocambille) [inline src/components/Register.jsx:101]: missing a confirmation for the password. even if you don't use it, show it ;)

**Review by rocambille — CHANGES_REQUESTED** [review]

File: src/components/Register.jsx (hunk 2 above)
Comment (tomriiich) [reply src/components/Register.jsx:101]: We added an input for the password confirmation. We will use it in the next sprint.

File: src/components/Home.jsx (hunk 1 above)
Comment (tomriiich) [reply src/components/Home.jsx]: Done

**Pull request merged**